view/frontend/web/js/components/BaseLoader.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="loader" v-if="loader">
+    <svg>
+      <circle cx="4" cy="4" r="4">
+        <animate
+          attributeName="opacity"
+          dur="1s"
+          values="0;1;0"
+          repeatCount="indefinite"
+          begin="0.1"
+        />
+      </circle>
+
+      <circle cx="18" cy="4" r="4">
+        <animate
+          attributeName="opacity"
+          dur="1s"
+          values="0;1;0"
+          repeatCount="indefinite"
+          begin="0.2"
+        />
+      </circle>
+
+      <circle cx="32" cy="4" r="4">
+        <animate
+          attributeName="opacity"
+          dur="1s"
+          values="0;1;0"
+          repeatCount="indefinite"
+          begin="0.3"
+        />
+      </circle>
+    </svg>
```

Comment:
please minify svg

Reply:
added svg-loader and add icon to assets + minify it and change styles to non scoped to get it in loader

Reply:
👍